index.html

```diff
@@ -1,5 +1,75 @@
 <!DOCTYPE html>
 <html lang="en">
+<<<<<<< HEAD
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link href='https://fonts.googleapis.com/css?family=Alex Brush' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Allura' rel='stylesheet'>
+    <title>Glimåkra Webbutvecklare Restaurangmenyer</title>
+
+</head>
+
+<body>
+
+    <header>
+        <section id="huvud" class="loose">
+            <img src="../lunchtimeweb-01_f73305aa-cd47-495d-9457-7a53dfcb022e_1024x1024@2x.png" alt="bild på nåt"
+                width="200px">
+            <h1>Slartibartfast Kulinarer</h1>
+    </header>
+    <main>
+        <ul>
+            <li>
+                <a href="menyer/exempel/index.html">Menyer </a>
+            <li>
+                <h1>Meny</h1>
+            </li>
+        </ul>
+
+        <h3>Välkommen till den kulinariska restauranger med de mest välutvecklade smaklökarna i denna galaxen. För en
+            extra upplevelse erbjuder vi även på poesi till eran rätt med den förfärligt dåliga ordbajsande Prostetnic
+            Vogon Jeltz</h3>
+        </section>
+
+        <section id="rätter" class="förvirrad">
+            <div class="mat">
+
+                <ul>
+                    <h3>Advanced Tea Substitute:</h3>Almost, but not quite, entirely unlike tea</li>
+                    <h3>Algolian Zylat Burger:</h3>A kind of a burger made from the the most unpleasant parts of a
+                    creature
+                    well known for its total lack of any pleasant parts</li>
+                    <h3>Starbix - the serious cereal:</h3>simply nasty but nourishing</li>
+                    <h3>Professional Sandwich from a professional sandwichmaker:</h3>Made from a perfectly normal beast,
+                    quite tasty</li>
+                    <h3>Pan Galactic Gargle Blaster:</h3>Best drink in Existence, effect similar to having you brains
+                    smashed out by a slice lemon wrapped around a large gold brick</li>
+                    <h3>Hagro Biscuit:</h3>Blue bread similar to an iced finger, topped with guacamole and "other" stuff
+                    (Don't ask)</li>
+                    <h3>Old Janx Spirit:</h3>This drink cause your head to fly, your tongue will lie, your eyes fry and
+                    you might die</li>
+                </ul>
```

Comment:
ul should only have li as children. There is a confusion with h3 and il. Currently it is <h3>blabla</h3>blibli</li>

```suggestion
                <ul>
                    <li><h3>Advanced Tea Substitute:</h3>Almost, but not quite, entirely unlike tea</li>
                    <li><h3>Algolian Zylat Burger:</h3>A kind of a burger made from the the most unpleasant parts of a
                    creature
                    well known for its total lack of any pleasant parts</li>
                    <li><h3>Starbix - the serious cereal:</h3>simply nasty but nourishing</li>
                    <li><h3>Professional Sandwich from a professional sandwichmaker:</h3>Made from a perfectly normal beast,
                    quite tasty</li>
                    <li><h3>Pan Galactic Gargle Blaster:</h3>Best drink in Existence, effect similar to having you brains
                    smashed out by a slice lemon wrapped around a large gold brick</li>
                    <li><h3>Hagro Biscuit:</h3>Blue bread similar to an iced finger, topped with guacamole and "other" stuff
                    (Don't ask)</li>
                    <li><h3>Old Janx Spirit:</h3>This drink cause your head to fly, your tongue will lie, your eyes fry and
                    you might die</li>
                </ul>
```